packages/@okta/vuepress-site/guides/sign-users-out/sign-out-of-okta/aspnetcore/remotesignout.md

```diff
@@ -34,46 +34,19 @@ using Microsoft.AspNetCore.Authentication.Cookies;
 using Okta.AspNetCore;
 ```
 
-After you sign users out of your app and out of Okta, you can also redirect users to a specific location. You need to whitelist the post sign-out URL in your Okta Application settings.
-
-Open the Okta Developer Console:
-
-<a href="https://login.okta.com/" target="_blank" class="Button--blue">Go to Console</a>
-
-1. Select **Applications**, and then pick your application.
-
-2. Select **General** and click **Edit**.
-
-3. Add the post sign-out URL in the  **Logout redirect URI** field, for example, `http://localhost:3000/account/postsignout`.
-
-4. Click **Save**.
-
-Then, you have to modify the Okta configuration in your application to also include the **Logout redirect URI**.
-Open the `Startup.cs` file and update the `ConfigureServices` method to include the `PostLogoutRedirectUri` in the Okta configuration:
+After you sign users out of your app and out of Okta, you can also redirect users to a specific location. Modify your `SignOut` action to include the action what you want to redirect to, for example, `Home`:
```

Comment:
changed "what" to "that"

After you sign users out of your app and out of Okta, you can also redirect users to a specific location. Modify your `SignOut` action to include the action that you want to redirect to, for example, `Home`: